Bridges/HttpKernel.php

```diff
@@ -63,23 +64,81 @@ public function bootstrap($appBootstrap, $appenv)
      */
     public function onRequest(ReactRequest $request, ReactResponse $response)
     {
-        if (null !== $this->application) {
-            try {
-                $syRequest = new SymfonyRequest();
-                $syRequest->headers->replace($request->getHeaders());
-                $syRequest->setMethod($request->getMethod());
-                $syRequest->server->set('REQUEST_URI', $request->getPath());
-                $syRequest->server->set('SERVER_NAME', explode(':', $request->getHeaders()['Host'])[0]);
-
-                $syResponse = $this->application->handle($syRequest);
-                $this->application->terminate($syRequest, $syResponse);
-
-                $headers = array_map('current', $syResponse->headers->all());
-                $response->writeHead($syResponse->getStatusCode(), $headers);
-                $response->end($syResponse->getContent());
-            } catch (\Exception $e) {
-                //
+        if (null === ($app = $this->application)) {
+            return;
+        }
+
+        $content = '';
+        $headers = $request->getHeaders();
+        $contentLength = isset($headers['Content-Length']) ? (int) $headers['Content-Length'] : 0;
+
+        $request->on('data', function($data)
+            use ($request, $response, &$content, $contentLength)
+        {
+            // read data (may be empty for GET request)
+            $content .= $data;
+
+            // handle request after receive
+            if (strlen($content) >= $contentLength) {
```

Comment:
Actually, I'm not sure about that. Depends not really on React, but rather on what the client is sending. If it sends `Connection: Keep-Alive` then I guess there's no "end". Also `Content-Type` says how the body is encoded and its encoding tells us then when the "end" is reached. I believe there's some research necessary, so I'm going to merge this anyway and create a new ticket for clients not sending `Content-Length` correctly.